eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -84,6 +84,8 @@ case class Commitments(channelVersion: ChannelVersion,
 
   def hasNoPendingHtlcs: Boolean = localCommit.spec.htlcs.isEmpty && remoteCommit.spec.htlcs.isEmpty && remoteNextCommitInfo.isRight
```

Comment:
This method is actually misleading, because the `remoteNextCommitInfo.isRight` statement will be true if the next signature only contains an `update_fee`. We should probably merge `hasNoPendingHtlcs` and `hasNoPendingFeeUpdate` (they are always called together anyway).

Reply:
Good catch

Reply:
Actually not really, it's called independently in `hasPendingOrProposedHtlcs`. But we can factor them in a new method that calls this one.